tools/kdc.json

```diff
@@ -26,9 +26,5 @@
       "name": "http"
```

Comment:
Since we no longer run the HTTP server, does this mean that this port is no longer required?

Reply:
Yup...good catch